compiler-core/src/docs/snapshots/gleam_core__docs__tests__markdown_code_from_standalone_pages_is_not_trimmed.snap

```diff
@@ -18,6 +18,7 @@ expression: "compile_with_markdown_pages(config, vec![], pages)"
          between multiple versions of the same package. -->
     <script src="./docs_config.js"></script>
     <link id="syntax-theme" rel="stylesheet" href="./css/atom-one-light.min.css?v=GLEAM_VERSION_HERE"/>
+    
```

Comment:
Could you configure all the tests to have them please 🙏

Reply:
These are not displayed because the project_name is blank.

Reply:
I am asking you to change that 🙏

Reply:
Yes! I'm asking you to add it to all of them 🙏  Thank you.

A test case for rendering not-for-hex would also be great.